includes/class-amp-story-post-type.php

```diff
@@ -1362,26 +1362,23 @@ public static function render_block_latest_stories( $attributes ) {
 	}
 
 	/**
-	 * Flushes rewrite rules if it hasn't been done yet after having AMP Stories Post type.
+	 * Add RSS feed link for stories.
+	 *
+	 * @since 1.2
 	 */
-	public static function maybe_flush_rewrite_rules() {
-		$current_rules = get_option( 'rewrite_rules' );
-
-		// If we're not using permalinks.
-		if ( empty( $current_rules ) ) {
-			return;
-		}
-
-		// Check if the rewrite rule for showing preview exists for different permalink settings.
-		$story_rules = array_filter(
-			array_keys( $current_rules ),
-			function( $rule ) {
-				return 0 === strpos( $rule, self::REWRITE_SLUG ) || false !== strpos( $rule, '/' . self::REWRITE_SLUG . '/' );
-			}
+	public static function print_feed_link() {
```

Comment:
Should this respect `current_theme_supports( 'automatic-feed-links' )`?

Reply:
@swissspidy Originally I did have this, but I reasoned myself out of it in https://github.com/ampproject/amp-wp/pull/2641#discussion_r294061433

Do you agree with my reasoning there?

Reply:
Another reason for why this theme support check doesn't make sense here is that it is not used by https://developer.wordpress.org/reference/functions/feed_links_extra/

Reply:
Ah, I see. Yes, makes sense to not have it then.